cmd/maya-apiserver/app/command/start.go

```diff
@@ -287,6 +287,13 @@ func (c *CmdStartOptions) setupMayaServer(mconfig *config.MayaConfig) error {
 
 	klog.Info("resources applied successfully by installer")
 
+	// clean older CASTemplates and RunTasks
+	cleanErr := install.SimpleInstaller().Clean()
+	if cleanErr != nil {
+		klog.Errorf("failed to clean old resources: %s", cleanErr.Error())
+		return errors.New("failed to clean old resources")
```

Comment:
Why are we returning error here? This error will kill the Maya-apiserver pod. IMO we should ignore this error, since we are logging.

Reply:
Done